static_src/components/tabnav.jsx

```diff
@@ -2,27 +2,31 @@
 import React from 'react';
 
 import classNames from 'classnames';
+import navStyles from 'cloudgov-style/css/components/nav.css';
+import createStyler from '../util/create_styler';
+
 
 export default class Tabnav extends React.Component {
   constructor(props) {
     super(props);
     this.props = props;
     this.state = { current: this.props.initialItem };
+    this.styler = createStyler(navStyles);
   }
 
   componentWillReceiveProps(nextProps) {
     this.setState({current: nextProps.initialItem});
   }
 
   render() {
-    var defaultClasses = ['nav', 'nav-tabs', 'nav-justified'];
+    var defaultClasses = [this.styler('nav'), this.styler('nav-tabs'), this.styler('nav-justified')];
```

Comment:
Nice work. It isn't well documented but you can actually pass in any number of classes to the `this.styler` function at a time. So this could be written as:
`this.styler('nav', 'nav-tabs', 'nav-justified')`